RxCocoa/Common/DelegateProxyType.swift

```diff
@@ -144,41 +186,42 @@ extension DelegateProxyType {
     ///
     ///     extension Reactive where Base: UISearchBar {
     ///
-    ///         public var delegate: DelegateProxy {
-    ///            return RxSearchBarDelegateProxy.proxyForObject(base)
+    ///         public var delegate: DelegateProxy<Base, UISearchBarDelegate> {
+    ///            return RxSearchBarDelegateProxy<Base>.proxyForObject(base)
```

Comment:
I think that we should make this `RxSearchBarDelegateProxy<Base>.proxy(for: base)`